### DISCUSSION TITLE
能否添加一个允许外部访问的功能

### DISCUSSION THREAD

**Discussion by lxrzlyr** [MetaCubeX/ClashMetaForAndroid#296] [Ideas]
手机开热点或者USB链接网络时，共享的设备也可以通过本机的VPN访问外网。iOS端的stash有这个功能，希望本应用也考虑支持这一功能。

**Comment by Bejukoro** [comment]
这个功能一直都是支持的，参见 https://wiki.metacubex.one/config/general/#_2
